README.md

```diff
@@ -1,75 +1,19 @@
-    │   │   │                                       predictions
-    │   │   ├── predict_model.py
-    │   │   └── train_model.py
-    │   │
-    │   └── visualization                       <- Scripts to create exploratory and results oriented visualizations
-    │       └── visualize.py
-    │
-    ├── .thoth.yaml                             <- Thoth's configuration file
-    ├── .aicoe-ci.yaml                          <- AICoE CI configuration file (https://github.com/AICoE/aicoe-ci)
-    │
-    ├── .github                                 <- GitHub configuration folder
-    │   ├── PULL_REQUEST_TEMPLATE               <- GitHub templates for pull requests
-    │   │
-    │   ├── ISSUE_TEMPLATE                      <- GitHub templates for issues
-    |       ├── {major|minor|patch}_release.md  <- If Khebut GitHub App Bot (https://github.com/apps/khebhut) is installed, the issue will trigger a major|minor|patch release.
-    │       |                                       The bot will open a Pull Request to update the CHANGELOG and fix the opened issue.
-    │       |                                       NOTE: only users that are allowed to release (a.k.a. maintainers specified in the .thoth.yaml file) should open the issue, otherwise bot will
-    │       |                                       reject them, commenting and closing the issue.
-    │       |                                       If AICoE CI GitHub App (https://github.com/apps/aicoe-ci) is installed, once the pull request is merged a new tag is created by the bot
-    │       |                                       and the pipeline to build and push image starts.
-    │       |
-    |       └── redeliver_container_image.md    <- If the tag exists and AICoE CI GitHub App (https://github.com/apps/aicoe-ci) is installed, the issue will retrigger the pipeline to build and
-    │                                               push image container image. NOTE: It should be used if the pipeline triggered with the {major|minor|patch}_release failed for any reason.
-    |
-    └── tox.ini                                 <- tox file with settings for running tox; see tox.readthedocs.io
+We fetch the CI test data from github and further store this to s3 storage. The main requirement here is to get test duration values. We can visualize the distribution of the test durations and based on the distribution type identified, we can find a point after which the test has a higher probability of failing.
 
---------
+## Feature Engineering
 
-<p><small>Project based on the <a target="_blank" href="https://drivendata.github.io/cookiecutter-data-science/">cookiecutter data science project template</a>. #cookiecutterdatascience</small></p>
+After fetching the data, we approximate the distributions of the test duration value and also check its goodness of fit for different tests. Based on the type of distribution identified, we can calculate the probability of the test failing.
```

Comment:
```suggestion
We then visualize the distribution of the test durations and based on the distribution type identified, we can find a point after which the test has a higher probability of failing. We approximate the distributions of the test duration value and also check its goodness of fit for different tests. Based on the type of distribution identified, we calculate the probability of the test failing.
```

Reply:
"check its goodness of fit for different tests" - should this be goodness of fit for different distributions? This paragraph is a little confusing, can we rephrase?